docs/contributing.md

```diff
@@ -33,6 +33,8 @@ related reports.
 * **Check that your issue does not already exist** in the [issue tracker].
 * **Make sure your issue is really a bug, and is not a support request or question** better suited for [Discussions]
 or [Discord].
+* **Try running your commands with** `--no-cache` **flag**
+* **Try clearing your cache with** `poetry cache clear --all PyPI` **and rerunning your commands**
```

Comment:
```suggestion
* **Try running your commands with the** `--no-cache` **flag**.
* **Try clearing your cache with** `poetry cache clear --all PyPI` **and rerunning your commands**.
```

Periods for consistency.

Apart from that, it feels a bit much **bold**, but since I do not have a better suggestion that is fine with me.

Reply:
Bolds are done for consistency with other bullet points in the list.